media/js/base/datalayer-trackscroll-init.es6.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrackScroll from './datalayer-trackscroll.es6';
+
+// Create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+window.Mozilla.TrackScroll = TrackScroll;
+
+window.addEventListener('scroll', TrackScroll.scrollListener);
```

Comment:
**Issue/suggestion:** Scroll events fire very rapidly, often dozens of times per second. This can have a knock on effect on performance in other areas of the page, so I'd suggest we at least de-bounce / throttle the event handler logic. Seeing as we're only recording 4 thresholds, `TrackScroll.scrolListener` could likely only need to fire every 200ms or so? Would need some testing to find the best value.

Reply:
**Nit:** Also suggest adding `useCapture` for better browser compatibility?
```suggestion
window.addEventListener('scroll', TrackScroll.scrollListener, false);
```

Reply:
200 seems like a safe start.